src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CertificateAccess/Model/DigitalCovidCertificate.swift

```diff
@@ -294,13 +294,13 @@ public struct Name: Codable, Equatable {
 
     public let familyName: String?
     public let givenName: String?
-    public let standardizedFamilyName: String
+    public let standardizedFamilyName: String?
```

Comment:
Is it really optional? as it should contain either of the component fnt or gnt

Reply:
Yes if we have only a given name and no family name this is a valid case so the given name should be assigned and the family name should be nil, this is why the tech spec says both technically can be nil, but logically we have to check that only one of them is nil